partiql-eval/src/test/kotlin/org/partiql/eval/internal/PartiQLEngineDefaultTest.kt

```diff
@@ -1385,6 +1333,73 @@ class PartiQLEngineDefaultTest {
             )
         ).assert()
 
+    @Test
+    @Disabled(
+        """
+            We currently do not have support for consolidating collections containing MISSING/NULL. The current
+            result (value) is correct. However, the types are slightly wrong due to the SUM__ANY_ANY being resolved.
```

Comment:
What should the correct type be then? Does this need corrected for just `t_c_sum`?

Reply:
Well, we don't have solidified type semantics when you see a literal NULL in a collection of tuples. Therefore, there is no correct type. I just know that the SUM that gets resolved takes in dynamic and returns a `null.null` rather than `null.int`.

Reply:
I see. Test comment makes sense to me now.